cmd/main.go

```diff
@@ -284,6 +284,10 @@ func main() {
 	var k8sClient *kubernetes.Clientset
 	var autoMigrationLeader leader.ElectorInterface
 
+	// Determine if IPXE artifact URLs need to be http
+	serverInfo := servers.New(Options.HTTPListenPort, swag.StringValue(port), Options.HTTPSKeyFile, Options.HTTPSCertFile)
+	generateInsecureIPXEURLs := serverInfo.HTTP != nil
```

Comment:
We shouldn't need to use this.

The next step runner pod will have the service cert mounted in and if someone is using http, we don't need to worry about messing about with the urls.

Reply:
Removed it, thank you!